pygotham/frontend/templates/home/index.html

```diff
@@ -1,12 +1,17 @@
 {% extends 'layouts/base.html' %}
 
+{% block extra_head %}
+<link href='https://fonts.googleapis.com/css?family=Aldrich' rel='stylesheet' type='text/css'>
```

Comment:
Nope, that was an oversight after grabbing the usage line from the Google Fonts website. Good catch.